source/platforms/git/localGetCommits.ts

```diff
@@ -36,11 +37,21 @@ export const localGetCommits = (base: string, head: string) =>
         parents: c.parents.split(" "),
       }))
 
-      done(realCommits)
+      commits.push(...realCommits)
```

Comment:
Looks like there’s variable shadowing going on here @orta. Given that you added a new commits array, I thought I’d flag it for you?

Reply:
> Looks like there’s variable shadowing going on here @orta. Given that you added a new commits array, I thought I’d flag it for you?

How this could be merged with this bug? Now the `danger local` is not working anymore as the `commits` variable is always overriding the commits list.

Reply:
I didn't notice, looks like that could get a test to verify that it doesn't break in the future then 👍🏻

Reply:
@orta I think it's probably missing some `no-shadow` lint rule.

Reply:
This issue was fixed on #1087.